lib/routes/jlu/phy/rcpy/index.ts

```diff
@@ -0,0 +1,31 @@
+import { Route } from '@/types';
+import { handler } from '../handler';
+
+export const route: Route = {
+    path: '/phy/rcpy/:category/:column',
```

Comment:
Parameter description should be [an object](https://docs.rsshub.app/joinus/new-rss/start-code#creating-route)